### PR TITLE
Improve cross hair of the speed space chart

### PR DESCRIPTION
**Pull request by flomonster**
Add white borders to the cross hair:

image

### CODE REVIEW

File: ui-speedspacechart/src/components/helpers/drawElements/reticle.ts [hunk 1]
```diff
@@ -181,14 +174,10 @@ export const drawCursor = ({ ctx, width, height, store }: DrawFunctionParams) =>
...
-
-  // Crosshair
-  ctx.beginPath();
-  ctx.moveTo(reticleX - RETICLE_LINE, reticleY);
-  ctx.lineTo(reticleX + RETICLE_LINE, reticleY);
-  ctx.moveTo(reticleX, reticleY - RETICLE_LINE);
-  ctx.lineTo(reticleX, reticleY + RETICLE_LINE);
-  ctx.stroke();
+  ctx.font = 'normal 12px IBM Plex Sans';
+  ctx.fillStyle = 'rgb(0,0,0)';
```
Comment (Yohh) [inline ui-speedspacechart/src/components/helpers/drawElements/reticle.ts]:
```suggestion
  ctx.fillStyle = BLACK.hex();
```

File: ui-speedspacechart/src/components/helpers/drawElements/reticle.ts [hunk 2]
```diff
@@ -181,14 +174,10 @@ export const drawCursor = ({ ctx, width, height, store }: DrawFunctionParams) =>
...
+  ctx.shadowOffsetY = 0;
+  ctx.shadowBlur = 0;
```
Comment (Yohh) [inline ui-speedspacechart/src/components/helpers/drawElements/reticle.ts]: why did you add those?

Comment (flomonster) [reply ui-speedspacechart/src/components/helpers/drawElements/reticle.ts]: I took this code from the top of the file. I don't think they're of any use and can be deleted at the same time.

**Review by Yohh — APPROVED** [review]
LGTM

> [!NOTE]
> IBM plex fonts are definitly not working in the GEV, we have to find a solution

File: ui-speedspacechart/src/components/helpers/drawElements/reticle.ts [hunk 3]
```diff
@@ -205,10 +192,32 @@ export const drawCursor = ({ ctx, width, height, store }: DrawFunctionParams) =>
...
+  ctx.lineWidth = 3;
+  ctx.strokeStyle = WHITE.hex();
+  ctx.lineCap = 'round';
+  ctx.moveTo(reticleX - RETICLE_LINE, reticleY);
+  ctx.lineTo(reticleX + RETICLE_LINE, reticleY);
+  ctx.moveTo(reticleX, reticleY - RETICLE_LINE);
+  ctx.lineTo(reticleX, reticleY + RETICLE_LINE);
+  ctx.stroke();
+
+  // Corsshair insides
```
Comment (SharglutDev) [inline ui-speedspacechart/src/components/helpers/drawElements/reticle.ts]:
```suggestion
  // Crosshair insides
```

**Review by SharglutDev — APPROVED** [review]
Lgtm and tested

File: ui-speedspacechart/src/components/helpers/drawElements/reticle.ts (hunk 3 above)
Comment (flomonster) [reply ui-speedspacechart/src/components/helpers/drawElements/reticle.ts]: Done
